third_party/abigen-celo/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.15.6-alpine@sha256:49b4eac11640066bc72c74b70202478b7d431c7d8918e0973d6e4aeb8b3129d2
+
+RUN apk add curl git gcc libc-dev linux-headers
+
+WORKDIR /app
+
+ADD go.mod .
+ADD go.sum .
+
+#RUN go build -mod=readonly -o /bin/abigen github.com/celo-org/celo-blockchain/cmd/abigen
+RUN go build -o /bin/abigen github.com/celo-org/celo-blockchain/cmd/abigen
```

Comment:
Try switching this to use a newer go image and also one that isn't using alpine and I think the rust problems might go away.

Reply:
I'd recommend 1.17.5 or 1.17.8

Reply:
Done.